### PR TITLE
Header footer v2

### LINKED ISSUE #360: Zooniverse Discuss and Project Discuss links are confusing
On my GZ bars project:

screen shot 2015-05-19 at 17 09 12

Should we change the name of one of them?

### PR DESCRIPTION
**Pull request by srallen**
- Updated styles for header and footer for consistency on other pages besides home. 
- Renamed Discuss in header and footer to Talk to fix #360

### CODE REVIEW

File: css/main-header.styl [hunk 1]
```diff
@@ -117,20 +116,18 @@
     margin: 0
 
 .main-nav-item
+  color: rgba(255,255,255,0.5) !important //ugh
```
Comment (brian-c) [inline css/main-header.styl]: Ehhh, let's fix the original problem (`.main-header a`, right?) instead of this.

**Comment by srallen** [comment]
Cool. I was hesitant with changing it and inadvertently changing something elsewhere like I did with the home page styles initially. I've updated and explicitly declared the anchor tag color styles.

**Comment by brian-c** [comment]
Nice, thanks. The header and footer should be pretty standalone. The rest of the site is kinda a mess. Might be time to bring in something like BEM.

One more tiny thing I just remembered, sorry: The account menu appears on when hovering over the space it takes up when visible. I _think_ you can quick-fix this by applying `pointer-events: none` to `.account-menu` and switching to `pointer-events: auto` when its parent is hovered.

**Comment by srallen** [comment]
Adding pointer-events fixed it.

Also agreed, BEM or something like it would be very helpful. The mess is partly my fault since I'm trying to get styles in quickly.

**Comment by brian-c** [comment]
Looks good!

**Pull request merged**